app/src/main/java/at/roteskreuz/stopcorona/screens/infection_info/InfectionInfoViewModel.kt

```diff
@@ -21,12 +21,9 @@ class InfectionInfoViewModel(
 ) : ScopedViewModel(appDispatchers) {
 
     fun observeInfectedContacts(): Observable<InfectedContactsViewState> {
-        return Observables.combineLatest(
-            infectionMessengerRepository.observeReceivedInfectionMessages(),
-            quarantineRepository.observeQuarantineState()
-        ).map { (messages, quarantineStatus) ->
+        return quarantineRepository.observeQuarantineState().map { quarantineStatus ->
             InfectedContactsViewState(
-                messages = messages,
+                messages = emptyList(),
```

Comment:
This emptyList() here seems unnecessary. We should check if we need to pass some additional messages, and if not we can remove the `messages` field from the `InfectedContactsViewState `.

Or add a TODO to be properly handled in the next MR.